internal/docker/cypress.go

```diff
@@ -44,19 +44,18 @@ func NewCypress(c cypress.Project, ms framework.MetadataService) (*CypressRunner
 
 // RunProject runs the tests defined in config.Project.
 func (r *CypressRunner) RunProject() (int, error) {
-	files := []string{
-		r.Project.Cypress.ConfigFile,
-		r.Project.Cypress.ProjectPath,
+	var files []string
+
+	if r.Project.RootDir != "" {
```

Comment:
What if I set `rootDir: node_modules`? Do we always consider that `rootDir` should have `cypress` folder and `cypress.json` ?

Reply:
@valentyn88 
Not sure this is an expected usecase, if you need your `node_modules` to be added next to your tests, you should use `rootDir: .`.

`cypress.configFile` should be relative to `rootDir`, if `rootDir` is not set, relative to the current directory (where `saucectl` is run).

Is there a use case I'm missing ?

Reply:
I also was testing the use-case `rootDir : some_folder`, where `some_folder` could have some additional files and folders for running your tests. With my use-case, we will fail, because `some_folder` won't have `cypress` folder and `cypress.json`. Maybe it's a case that we don't need to worry about.

Reply:
cypress (and all related files) need to be somewhere (no matter how deeply nested) under the rootDir.
If they are not, then rootDir is not root.